satellite/nodeselection/node.go

```diff
@@ -19,6 +19,8 @@ import (
 	"storj.io/storj/shared/location"
 )
 
+var errTagsNotFound = errs.New("tags not found")
```

Comment:
errs is going to attach a stacktrace, i think if we're going to avoid errs, then we should use the stdlib error.New

Reply:
oh wait you've just cherry picked things. i guess this is a comment on the original review

Reply:
Yes, but generally impact is very small